libs/common-system/src/main/java/org/codice/ddf/configuration/DictionaryMap.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration;
+
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.Dictionary;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * A HashMap based Dictionary for use with OSGI methods requiring a Dictionary
+ *
+ * @param <K>
+ * @param <V>
+ */
+public class DictionaryMap<K, V> extends Dictionary<K, V> implements Map<K, V>, Serializable {
```

Comment:
This class lets us interact with the osgi interfaces without the need to use the Hashtable which is synchronized.

Reply:
I don't really like the name so if you have a better one let me know